### PR TITLE
Updates to Berks page

### PR DESCRIPTION
**Pull request by jeffrummel**

### CODE REVIEW

File: content/berks/_index.md [hunk 1]
```diff
@@ -50,7 +54,11 @@ creditRole="Capitol Reporter"
 creditEmail="heather@spotlightpa.org"
 creditBio="Interested in learning more? Contact Heather Adams, Berks County Project Lead, at [heather@spotlightpa.org](mailto:heather@spotlightpa.org)"
 >}}
-## A Novel Approach to Reinvigorating Local News
+
+## [Read our editors statement about this study](https://www.spotlightpa.org/news/2023/09/reading-pennsylvania-news-berks-county-spotlight-pennsylvania/)
+
+
+### A Novel Approach to Reinvigorating Local News
```
Comment (carlmjohnson) [inline content/berks/_index.md:61]: Should this be H2 or H3?

Comment (jeffrummel) [reply content/berks/_index.md:61]: Structurally it should be h3, I chose h2 for the aesthetics, but we're going with the link at the top (line 44)

**Comment by jeffrummel** [comment]
@carlmjohnson Please hold on your review, I have more changes coming

File: content/berks/_index.md [hunk 2]
```diff
@@ -40,31 +40,42 @@ hed="at West Lawn-Wyomissing Hills Library"
...
+actionText = "Continue Reading →"
 
-credit = """
-{{<featured/credit-long
-shortcode="{{<supplemental>}}"
->}}"""
+[[blocks.entries]]
+hed = "Take the Berks County News Consumption and Media Coverage Survey"
+actionUrl = "https://www.surveymonkey.com/r/SpotlightPABerksProject"
+actionText = "Begin survay →"
```
Comment (carlmjohnson) [inline content/berks/_index.md:60]: Spelling: survey

File: content/berks/_index.md [hunk 3]
```diff
@@ -40,31 +40,42 @@ hed="at West Lawn-Wyomissing Hills Library"
 dateVerbose="Wednesday February 21 – 5:30-7:30pm"
 registration="https://spotlight-pa-berks.ticketleap.com/westlawnwyohillssession/"
 >}}
+
+<h3><a href="https://www.spotlightpa.org/news/2023/09/reading-pennsylvania-news-berks-county-spotlight-pennsylvania/" class="hover:underline hover:text-yellow">Read our editors statement about this study</a></h3>
```
Comment (carlmjohnson) [inline content/berks/_index.md:44]: Can you underline this before hover? It doesn't quite look like a link to me.

**Review by carlmjohnson — APPROVED** [review]
LGTM.

**Pull request merged**